### PR TITLE
add internal/errors/redis test

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

This PR includes the following changes:
  - add unit test for `internal/errors/redis`
  - add comment for each function of `internal/errros/redis.go`
 
**Grammar check has passed.**

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/errors/redis.go [hunk 1]
```diff
@@ -19,56 +19,72 @@ package errors
...
-	ErrRedisAddrsNotFound = New("addrs not found")
+	// ErrRedisAddrsNotFound generates a new error of address not found.
+	ErrRedisAddrsNotFound = New("error redis addrs not found")
 
-	ErrRedisConnectionPingFailed = New("error Redis connection ping failed")
+	// ErrRedisConnectionPingFailed generates a new error of Redis connection ping failed.
+	ErrRedisConnectionPingFailed = New("error redis connection ping failed")
 )
 
+// ErrRedisNotFoundIdentity represents a struct that includes err and has method for Redis error handling.
```
Comment (hlts2) [inline internal/errors/redis.go:69]:
```suggestion
// ErrRedisNotFoundIdentity represents a struct that includes err and has a method for Redis error handling.
```

**Review by hlts2 — DISMISSED** [review]
thank you
lgtm

File: internal/errors/redis_test.go [hunk 2]
```diff
@@ -239,3 +848,70 @@ func TestErrRedisNotFoundIdentity_Unwrap(t *testing.T) {
...
+	}
+	defaultCheckFunc := func(w want, got bool) error {
+		if !reflect.DeepEqual(got, w.want) {
+			return Errorf("got: \"%#v\",\n\t\t\t\twant: \"%#v\"", got, w.want)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "return false when err is nil",
```
Comment (kevindiu) [inline internal/errors/redis_test.go:875]:
```suggestion
			name: "return false when err is not ErrRedisNotFoundIdentity ",
```

**Review by hlts2 — DISMISSED** [review]
thank you :bow:
LGTMed

**Pull request merged**